src/renderer/components/widgets/Follow.js

```diff
@@ -0,0 +1,45 @@
+import {FaBell} from "react-icons/fa/index";
+import React, {Component} from "react";
+import {Button} from "react-bootstrap";
+
+class Follow extends Component {
+    constructor() {
+        super()
+        this.state = {followers: 0}
+    }
+
+    componentDidMount() {
+        fetch('https://api.openhive.network', {
+            method: 'POST',
+            /*headers: {'Content-Type': 'application/json'},*/
+            body: JSON.stringify({
+                jsonrpc:"2.0",
+                method:"follow_api.get_follow_count",
+                params:{"account":this.props.user},
```

Comment:
This section needs to support RefLink formatting. Line [18](https://github.com/vaultec81/blasio-app/pull/3/files#diff-f2aa2c551db7ab8b6cd185611edc94c7R18) is of primarily interest. 

Also could you use dhive, or hivejs for API handling as opposed to using a raw fetch? (primarily thinking ahead here when we have support for a global settings menu, and a way to change hive API. Which I would like to have API settings globally enforced across the entire code base)